### PR TITLE
[tensorexpr] aten::Linear external call

### PR DESCRIPTION
**Pull request by IvanKobzarev**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* __->__ #64751

Differential Revision: [D30842102](https://our.internmc.facebook.com/intern/diff/D30842102)

### CODE REVIEW

**Comment by IvanKobzarev** [comment]
@ivankobzarev has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D30842102).

**Comment by IvanKobzarev** [comment]
@ivankobzarev has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D30842102).

**Comment by IvanKobzarev** [comment]
@ivankobzarev has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D30842102).

**Comment by IvanKobzarev** [comment]
@ivankobzarev has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D30842102).

File: torch/csrc/jit/tensorexpr/kernel.cpp [hunk 1]
```diff
@@ -2523,6 +2523,20 @@ Tensor tensorexpr::computeOperandValue(
...
+      // linear = inputs[0] @ inputs[1] + inputs[2]
+      // addmm = beta(inputs[3]) * inputs[0] + alpha(inputs[4]) * inputs[1] @
+      // inputs[2]
+      std::vector<ArgValue> addmmInputs;
+      addmmInputs.reserve(5);
+      addmmInputs.push_back(inputs[2]);
+      addmmInputs.push_back(inputs[0]);
+      addmmInputs.push_back(inputs[1]);
+      // Using int64_t as computeAddMM supports only int64_t scalars now
+      addmmInputs.push_back(1l); // beta
```
Comment (navahgar) [inline torch/csrc/jit/tensorexpr/kernel.cpp:2536]: s/1l/1ll

File: torch/csrc/jit/tensorexpr/kernel.cpp [hunk 2]
```diff
@@ -2523,6 +2523,20 @@ Tensor tensorexpr::computeOperandValue(
...
+      addmmInputs.push_back(1l); // alpha
```
Comment (navahgar) [inline torch/csrc/jit/tensorexpr/kernel.cpp:2537]: s/1l/1ll

**Review by navahgar — COMMENTED** [review]
What about a test for this?

**Comment by IvanKobzarev** [comment]
@ivankobzarev has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D30842102).

**Comment by IvanKobzarev** [comment]
@ivankobzarev has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D30842102).

**Comment by IvanKobzarev** [comment]
@ivankobzarev has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D30842102).

**Comment by IvanKobzarev** [comment]
@ivankobzarev has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D30842102).

**Comment by IvanKobzarev** [comment]
@ivankobzarev has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D30842102).

**Comment by IvanKobzarev** [comment]
@ivankobzarev has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D30842102).

**Review by navahgar — APPROVED** [review]
LGTM

**Comment by IvanKobzarev** [comment]
@ivankobzarev has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D30842102).

**Pull request closed without merging**